.github/workflows/test-playwright.yml

```
@@ -0,0 +1,170 @@
+      - name: Override ref_name
+        id: ref_name
+        run: |
+          if [ -n "${{ github.event.inputs.ref_name }}" ]; then
+            echo "ref_name=${{ github.event.inputs.ref_name }}" >> $GITHUB_OUTPUT
+          elif [ -n "${{ github.event.workflow_run.head_branch }}" ]; then
+            echo "ref_name=${{ github.event.workflow_run.head_branch }}" >> $GITHUB_OUTPUT
+          else
+            echo "ref_name=${{ github.ref_name }}" >> $GITHUB_OUTPUT
+          fi
+ 
+      - name:  Start CANChat Environment
```

YAMLlint (1.38.0)

[warning] too many spaces after colon

(colons)